examples/bri-3/src/bri/zeroKnowledgeProof/capabilities/verifyAnchorHash/verifyAnchorHashCommand.handler.ts

```diff
@@ -23,6 +24,8 @@ export class VerifyAnchorHashCommandHandler
         command.inputForProofVerification,
```

Comment:
What is inputForProofVerification? We are supposed to verify anchor hash here. Naming is a confusing.

Reply:
And what is publicInputForProofVerification?

Reply:
`inputForProofVerification` are any type of inputs that is required to verify the on-chain anchor hash. In our current iteration, we only need the State object to verify the hash. But in future iteration, to allow the hash to be created and verified using zk-circuits and proofs, the variable is called `inputForProofVerification`.

In the BPI Integration section of the Standard, it mentions that the following items are required for proof verification `a ZK proof, associated public input data, proof-verification key, a verification endpoint for the ZK proof verification, and a predicate as a human-readable version of the ZK Proof.`

The param `publicInputForProofVerification` has been added as a placeholder for future iterations.

Reply:
@ognjenkurtic want a todo or a rename?

Reply:
I am fine we leave it like this after @biscuitdey explanation.